source/common/stats/symbol_table_impl.h

```diff
@@ -523,22 +541,11 @@ class StatNameList {
   SymbolTable::StoragePtr storage_;
 };
 
-// Helper class for constructing hash-tables with StatName keys.
-struct StatNameHash {
-  size_t operator()(const StatName& a) const { return a.hash(); }
-};
-
-// Helper class for constructing hash-tables with StatName keys.
-struct StatNameCompare {
-  bool operator()(const StatName& a, const StatName& b) const { return a == b; }
-};
-
 // Value-templatized hash-map with StatName key.
-template <class T>
-using StatNameHashMap = absl::flat_hash_map<StatName, T, StatNameHash, StatNameCompare>;
+template <class T> using StatNameHashMap = absl::flat_hash_map<StatName, T>;
 
 // Hash-set of StatNames
-using StatNameHashSet = absl::flat_hash_set<StatName, StatNameHash, StatNameCompare>;
+using StatNameHashSet = absl::flat_hash_set<StatName>;
```

Comment:
you should consider if you still want to use these aliases or not.  Either is fine, just posing it as a question

Reply:
My inclination is to keep them. I am curious: is there any runtime benefit to having this default hasher, compared to specifying the functor? Or it's just nicer because the type is simpler to type?

I guess one functional difference is that we're using absl's default hash algorithm rather than xxhash, which maybe is tuned for the swiss tables better?